### ISSUE TITLE
Message forwarding

### ISSUE THREAD

**Issue by dbrgn** [threema-ch/threema-web#3]
Possible approaches:

- Show a "forward" entry in the context menu of a message
- Enable drag&drop of messages to other conversations or contacts

**Comment by leeps** [comment]
Related:
How would I move the complete message history from one contact to another (e.g. if someone switched IDs)?
It would be nice if there was a way without manually moving each message.

**Comment by dbrgn** [comment]
@leeps That will probably not be possible any time soon. A conversation is currently bound to a Threema ID. This issue is about sending another message with the same content to a different ID.

**Comment by leeps** [comment]
You're right, not as related as I thought - I don't want to send past messages.
